indico/modules/events/operations.py

```diff
@@ -42,3 +42,10 @@ def delete_reference_type(reference_type):
     db.session.delete(reference_type)
     db.session.flush()
     logger.info('Reference type "%s" deleted by %s', reference_type, session.user)
+
+
+def create_event_references(event, data):
+    event.populate_from_dict(data)
```

Comment:
`event.references = data['references']` - this is not a generic "modify event" function!